plugin.json

```diff
@@ -64,6 +64,6 @@
         "default": ""
       }
     ],
-    "footer": "Use this webhook URL to set up the Jira integration. See [documentation](https://about.mattermost.com/default-jira-plugin) to learn more.\n\n`https://SITEURL/plugins/jira/webhook?secret=WEBHOOKSECRET&team=TEAMURL&channel=CHANNELURL`"
+    "footer": "Use this webhook URL format to set up the Jira integration. See [documentation](https://about.mattermost.com/default-jira-plugin) to learn more.\n\n Jira 2.1 use: `https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET` \n\n For per channel webhook configuration use this format: `https://SITEURL/plugins/jira/webhook?secret=WEBHOOKSECRET&team=TEAMURL&channel=CHANNELURL"
```

Comment:
Sorry that I'm nitpicking .. but 

1. does `Jira 2.1 use:` add value since these changes will only appear in 2.1 and later?
2. is `per channel webhook configuration` obvious to a System Admin?

Reply:
@jasonblais Not nitpicking, no prob.  I was going to refer to the "per channel webhook" in the plugin documentation in the area dedicated to webhooks (because it's no longer needed, but optional).  We need to explain the subtlety, but I'm second guessing including the "per channel webhooks" now because it muddles things up.  Perhaps I should just strip it down to the new mega webhook URL in the footer, and refer everything else to the docs?

Reply:
Yeah, I'm 0/5. If there is no value having the old webhook URL format on the page, we can leave it out, given the link to docs for more details.